src/main/java/org/cactoos/scalar/AndWithIndex.java

```diff
@@ -42,15 +42,17 @@
  * works, but with an index provided for each item:</p>
  *
  * <pre>
- * new UncheckedScalar<>(
- *     new AndWithIndex(
- *         new IterableOf<>("Mary", "John", "William", "Napkin"),
- *         new BiFuncOf<>((text, index) ->
- *             System.out.printf("| idx #%d: name: %s ", index, text), true)
- *     )
- * ).value();
- * // will print "| idx #0: name: Mary | idx #1: name: John |
- * idx #2: name: William | idx #3: name: Napkin " to console
+ * {@code
+ *   new UncheckedScalar<>(
+ *       new AndWithIndex(
+ *           new IterableOf<>("Mary", "John", "William", "Napkin"),
+ *           new BiFuncOf<>((text, index) ->
+ *               System.out.printf("| idx #%d: name: %s ", index, text), true)
+ *       )
+ *   ).value();
```

Comment:
@Isammoc Please, correct this indentation.

Reply:
@paulodamaso Done